### ISSUE TITLE
[BUG] - GraphRAG dependencies not installed. GraphRAG retriever pipeline will not work properly

### ISSUE THREAD

**Issue by Harvester62** [Cinnamon/kotaemon#321]
### Description

The GraphRag functionality is not working due to apparently missing libraries. See attached screenshot of the first startup (there are a few errors about Powershell). 

In fact, trying to run the research in an already loaded file into GraphRag, it returns an ERROR box message in the web UI.

![kotaemon-graphrag-errors](https://github.com/user-attachments/assets/f116500b-eea6-47b4-87b1-08720508f310)

![kotaemon-graphrag-error](https://github.com/user-attachments/assets/5f95279f-0bb1-410f-b1c8-7fe9175c7686)

### Reproduction steps

```bash
Current environment: 

Windows 10, 32GB RAM
Nvidia GTX1070, 8GB VRAM
Python 3.10.6

1. Go to 'kotaemon-app\scripts'
2. Run 'run_windows.bat'
3. Load a doc in GraphRag
4. Search in the doc
5. See error
```

### Browsers

Firefox

### OS

Windows

**Comment by tzzed** [comment]
I have the same issue on macos

**Comment by Lee-Ju-Yeong** [comment]
I have the same issue on macos

**Comment by Lee-Ju-Yeong** [comment]
> I have the same issue on macos

I encountered an issue during the setup and execution of GraphRAG indexing and retrieval pipelines in the Kotaemon environment. After following the steps below to resolve dependency issues, I managed to get rid of the "GraphRAG dependencies not installed" error, but I am now facing a new error related to reading .parquet files from an incorrect directory path.

Steps Taken to Resolve the Issue:
Deactivate and Remove Previous Virtual Environment:
conda deactivate
conda remove -n kotaemon --all

Create and Activate a New Virtual Environment:
conda create -n kotaemon python=3.10
conda activate kotaemon

Install Previous Version of GraphRAG:
pip install graphrag==0.3.5

Install Required Dependencies:
pip install -e "libs/kotaemon[all]"
pip install -e "libs/ktem"

Check for Dependency Conflicts:
pip check

Manually Adjust Package Versions:
pip install aiofiles==23.2.1
pip install environs==9.5.0
pip install tenacity==8.2.3

Run the indexing process: The system proceeded with indexing, and everything seemed fine, as I did not receive the "GraphRAG dependencies not installed" message anymore.


but after that, During the execution of the retrieval pipeline, I encountered the following error:
`
FileNotFoundError: Parquet file not found: /Users/zooyong/Documents/Kotaemon/ktem_app_data/user_data/files/graphrag/bb84bca3-2bd0-4f70-936b-a4c095b1ab9f/output/create_final_nodes.parquet`
